jujugui/static/gui/src/app/components/usso-login-link/_usso-login-link.scss

```diff
@@ -0,0 +1,26 @@
+.usso-login {
+  position: relative;
+
+  &__notification {
+    background: #fff;
```

Comment:
We should avoid using hex colours and use Scss variables instead. To help constancy. Please use `$white`.